### PR TITLE
modified the script for creating trackDb.txt

### PR DESCRIPTION
**Pull request by keyvanelhami**
In this PR, the script for creating the trackDb.txt is modified to also include the bigwig files that were recently added to this repo. An additional for loop is added to include the bigwig files. Furthermore, the README files is updated and the new trackDb.txt is added

### CODE REVIEW

**Comment by keyvanelhami** [comment]
@hassanfa I have tested the script and it worked for me. Please have a look

**Comment by hassanfa** [comment]
@keyvanelhami  could add a bit to PR text body what each PR are? I understand there is a default PR template, but that's for new bed files. I'm looking into script now.

File: README.md [hunk 1]
```diff
@@ -364,19 +364,38 @@ Create trackDb.txt
...
+
+#Remove the ".bw" extension in the end of the file and loop through filename
+for f in `ls ucsc_track_hub/hg19/*bw | rev | cut -d"." -f 2- | rev`;
+do
+  i=`basename $f`;
+  n=`echo $i | cut -d"_" -f 2`;
+  tr=`echo $i | cut -d"_" -f 1-2`;
+  lng=`echo $i | cut -d"_" -f 1-3`;
+  echo "track "$tr >> ucsc_track_hub/hg19/trackDb.txt;
+  echo "shortLabel CG"$n "BigWeb" >> ucsc_track_hub/hg19/trackDb.txt;
```
Comment (hassanfa) [inline README.md:392]: Do you mean bigwig?

File: README.md [hunk 2]
```diff
@@ -364,19 +364,38 @@ Create trackDb.txt
...
+  echo "type BigWeb" >> ucsc_track_hub/hg19/trackDb.txt;
```
Comment (hassanfa) [inline README.md:393]: I guess you mean bigwig here as well?

File: README.md [hunk 3]
```diff
@@ -364,19 +364,38 @@ Create trackDb.txt
...
+  echo "longLabel "$lng "BigWeb" >> ucsc_track_hub/hg19/trackDb.txt;
```
Comment (hassanfa) [inline README.md:394]: same here as well. I assume it is bigwig?

File: README.md [hunk 4]
```diff
@@ -364,19 +364,38 @@ Create trackDb.txt
...
 done
+
+#Remove the ".bw" extension in the end of the file and loop through filename
+for f in `ls ucsc_track_hub/hg19/*bw | rev | cut -d"." -f 2- | rev`;
+do
+  i=`basename $f`;
+  n=`echo $i | cut -d"_" -f 2`;
+  tr=`echo $i | cut -d"_" -f 1-2`;
+  lng=`echo $i | cut -d"_" -f 1-3`;
+  echo "track "$tr >> ucsc_track_hub/hg19/trackDb.txt;
```
Comment (hassanfa) [inline README.md:391]: This won't work. There will tracks with exactly same name. `track` field in trackDb.txt has to be unique unfortunately.

File: README.md (hunk 2 above)
Comment (keyvanelhami) [reply README.md:393]: Yes, I have corrected the typo

File: README.md (hunk 3 above)
Comment (keyvanelhami) [reply README.md:394]: Yes, I have corrected the typo

File: README.md (hunk 4 above)
Comment (keyvanelhami) [reply README.md]: I added an extra extension, please look at the latest commits

File: README.md (hunk 1 above)
Comment (keyvanelhami) [reply README.md]: Yes, I have corrected the typo

**Comment by hassanfa** [comment]
Seems like new trackDb.txt didn't pass the tests. Could you please look into it again?

**Comment by keyvanelhami** [comment]
@hassanfa checked the travis logs and found the below message. I looked into `hub.txt` and should the BigWig format somehow be presented there??

```
Unsupported type 'BigWig' in hub ucsc_track_hub/hub.txt genome hg19 track gicfdna_3.1_BigWig
The command "utilities/ucsc_utils/linux.x86_64/hubCheck -checkSettings -verbose=3 -noTracks -printMeta ucsc_track_hub/hub.txt" exited with 1.
```

**Comment by hassanfa** [comment]
The line I added to `.travis.yml` checks `hub.txt`. It checks all the contents of the `trackDb.txt` and accompanying files and paths.

From what I can see on UCSC documentation, seems like the keyword `BigWig` is wrong and it should be `bigWig`.

**Comment by hassanfa** [comment]
Also, you could check the hub data yourself before pushing to the repo:

If you're on Hasta:
`utilities/ucsc_utils/linux.x86_64/hubCheck -checkSettings -verbose=3 -noTracks -printMeta ucsc_track_hub/hub.txt` 

If you're on your Mac OS X:
`utilities/ucsc_utils/macOSX.x86_64/hubCheck -checkSettings -verbose=3 -noTracks -printMeta ucsc_track_hub/hub.txt`

**Comment by keyvanelhami** [comment]
@hassanfa It finally passed. It was supposed to be `bigWig`

**Pull request merged**